### PR TITLE
monte carlo simulations

### PR DESCRIPTION
**Pull request by jaiswalpuru**

### CODE REVIEW

**Comment by jaiswalpuru** [comment]
Any updates ?

File: math/montecarlo/estimation_e.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"math"
+	"math/rand"
+	"time"
+)
+
+const (
+	e = math.E
+)
+
+func estimation_e() {
```
Comment (tjgurwara99) [inline math/montecarlo/estimation_e.go:14]: This is an untestable implementation. Please write code that you can easily test. Refer to [How to contribute](https://github.com/TheAlgorithms/Go/blob/master/CONTRIBUTING.md) and look at some other code within this repo to see how to do it.

Comment (jaiswalpuru) [reply math/montecarlo/estimation_e.go:14]: Sure, thank you.

**Comment by tjgurwara99** [comment]
Please take a look at our new [contribution guidelines](https://github.com/TheAlgorithms/Go/blob/d2f12849df76203d80762ceca5105531d3424384/CONTRIBUTING.md) and make changes accordingly. Currently this PR is not up to standard. If no changes are made in the next two weeks, I will consider this PR to be stale and will close this PR then.

**Pull request closed without merging**